src/backend/revoker/subScan.ts

```diff
@@ -12,69 +12,141 @@ const SUBSCAN_MAX_ROWS = 100;
 const QUERY_INTERVAL_MS = 1000;
 const BLOCK_RANGE_SIZE = 100_000;
 
-const eventsApi = `https://${subscan.network}.api.subscan.io/api/scan/events`;
+const eventsListURL = `https://${subscan.network}.api.subscan.io/api/v2/scan/events`;
+const eventsParamsURL = `https://${subscan.network}.api.subscan.io/api/scan/event/params`;
 const headers = {
   'X-API-Key': subscan.secret,
 };
 
-export interface EventsResponseJson {
+/**
+ * Structure of SubScan responses from `/api/v2/scan/events`.
+ */
+export interface EventsListJSON {
+  code?: number;
   data: {
     count: number;
-    events: Array<{
-      params: string;
-      block_num: number;
-      block_timestamp: number;
-      extrinsic_hash: HexString;
-    }> | null;
+    events:
+      | {
+          block_timestamp: number; // UNIX-time in seconds
+          event_id: string;
+          event_index: string;
+          extrinsic_hash: string;
+          extrinsic_index: string;
+          finalized: true;
+          id: number;
+          module_id: string;
+          phase: number;
+        }[]
+      | null;
   };
+  generated_at?: number;
+  message?: string;
+}
+
+/**
+ * Structure of SubScan responses from `/api/scan/event/params`.
+ */
+export interface EventsParamsJSON {
+  code: number;
+  data: {
+    event_index: string;
+    params: {
+      name?: string;
+      type?: string;
+      type_name: string;
+      value: unknown;
+    }[];
+  }[];
+  generated_at: number;
+  message: string;
 }
 
 export async function getEvents({
   fromBlock,
   row = SUBSCAN_MAX_ROWS,
   ...parameters
 }: {
-  module: string;
-  call: string;
+  module: string; // Pallet name
+  event_id: string; // Event emitted
```

Comment:
Just because SubScan uses that name and it is needed on the request. It spares a transformation of the `parameters`. 

Should I change it to camelCase?  🐍🔄🐫

Reply:
It’s a good practice to maintain a code style, and sometimes even enforced by linting rules. They are not disabled in our case for the benefit of destructuring of API data, but we should try to limit this. Please change it to camelCase.

Reply:
Ok, will do.

Reply:
Done. :)